ExploreASL_Master.m

```diff
@@ -1,37 +1,54 @@
-function [x] = ExploreASL_Master(DataParPath, ProcessData, SkipPause, iWorker, nWorkers, iModules)
-%ExploreASL_Master ExploreASL pipeline master wrapper calling the individual pipeline modules
+function [x] = ExploreASL_Master(varargin)

```

Comment:
Seeing that the input parameters changed a bit - have you checked that this change is reflected in all functions that call the master script (ExploreASL_Master or ExploreASL)? There should be at least one, right - xASL_qc_Test...

Reply:
Not yet. I wanted to wait for feedback, so we don't have to change these scripts multiple times. I can do it know if you're happy with the way the Master scripts looks right now.

Reply:
Hopefully fixed all in 8c1269d.